modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v7_0_4/ResourcePermissionUpgradeProcess.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.internal.upgrade.v7_0_4;
+
+import com.liferay.dynamic.data.mapping.model.DDMFormInstance;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.dao.orm.common.SQLTransformer;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Alberto Sousa
+ */
+public class ResourcePermissionUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
+				SQLTransformer.transform(
+					StringBundler.concat(
+						"select resourcePermissionId, ctCollectionId from ",
+						"ResourcePermission where name = '", _CLASS_NAME_POLLS,
+						"'")));
```

Comment:
@abesousa please follow the same order as [portal-tables.sql](https://github.com/liferay/liferay-portal/blob/620d8e89d9a7e5b5789073967027ad259b62a563/sql/portal-tables.sql#L1240-L1254) and put  `ctCollectionId` first